components/renku_data_services/storage/rclone.py

```diff
@@ -363,7 +416,7 @@ class RCloneProviderSchema(BaseModel):
     @property
     def required_options(self) -> list[RCloneOption]:
         """Returns all required options for this provider."""
-        return [o for o in self.options if o.required]
+        return [o for o in self.options if o.required and not o.sensitive]
```

Comment:
issue: could you write a comment on why sensitive required options should not be returned here?

Reply:
I improved the code so that this is no longer necessary.